### PR TITLE
add_blocktensor_9

### PR DESCRIPTION
**Pull request by mganahl**
Next PR for block sparsity

### CODE REVIEW

File: tensornetwork/block_sparse/block_tensor.py [hunk 1]
```diff
@@ -1613,3 +1620,120 @@ def tensordot(tensor1: BlockSparseTensor,
       order=left_order + right_order,
       check_consistency=False)
   return res
+
+
+def svd(matrix: BlockSparseTensor,
```
Comment (Thenerdstation) [inline tensornetwork/block_sparse/block_tensor.py]: Unit tests please. :)

Comment (mganahl) [reply tensornetwork/block_sparse/block_tensor.py]: sry, I had accidentally added the tests to the next PR block_tensor_test.py!

Comment (mganahl) [reply tensornetwork/block_sparse/block_tensor.py]: there now!

File: tensornetwork/block_sparse/block_tensor.py [hunk 2]
```diff
@@ -1613,3 +1620,120 @@ def tensordot(tensor1: BlockSparseTensor,
...
+        full_matrices: Optional[bool] = True,
+        compute_uv: Optional[bool] = True,
+        hermitian: Optional[bool] = False) -> Any:
+  """
+  Compute the singular value decomposition of `matrix`.
+  The matrix if factorized into `u * s * vh`, with 
+  `u` and `vh` the left and right eigenvectors of `matrix`,
```
Comment (Thenerdstation) [inline tensornetwork/block_sparse/block_tensor.py]: singular vectors.

File: tensornetwork/block_sparse/block_tensor.py [hunk 3]
```diff
@@ -1613,3 +1620,120 @@ def tensordot(tensor1: BlockSparseTensor,
...
+  and `s` its singular values.
+  Args:
+    matrix: A matrix (i.e. a rank-2 tensor) of type  `BlockSparseTensor`
```
Comment (Thenerdstation) [inline tensornetwork/block_sparse/block_tensor.py]: we prefer to use "order"

**Pull request merged**